operational/fnms/schedule_fnms_reports/README.md

```diff
@@ -1,46 +1,66 @@
-## Schedule FlexNet Manager report
+# Schedule FlexNet Manager report
+
+## What it does
 
-### What it does
 This policy will run a FlexNet Manager report (Custom view) and send the result via email.
 The policy is a report only policy, no action is taken during the Policy Escalation.
 
 The report / Mail output looks like this:
 ![Alt text][emailoutput]
 
 Current limitations:
-- This only works with the FNMS Cloud
+
+- This only works with the FNMS Cloud or a onPrem installation available on the internet.
+
 - Output is limited to max 100.000 rows.
 
-### Pre-reqs
+## Pre-reqs
+
 - FlexNet Manager
 - The following RightScale Credentials
-  - 'FNMS_API_Token'
+  - FlexNet Manager Cloud
+    - 'FNMS_API_Token'
+
+  - FlexNet Manager onPrem (on the internet)
+    - 'FNMS_API_Username'
+    - 'FNMS_API_Password'
 
-### Installation
+## Installation
 
-#### How to setup FlexNet Manager Custom View for this policy:
+### How to setup FlexNet Manager Custom View for this policy:
 
 1. Create a custom view in FlexNet manager that could look like this:
 ![Alt text][FNMSReport]
 Once saved note the report number in thr URL field :
 ![Alt text][ReportNumber] you need it when activating the Policy
 
-1. Setup the API Token in FNMS:
+1. Setup the API Token in FNMS Cloud:
     1. On the Account page - Select Create Account -> Service Account and fill in the form
 
         ![Alt text][CreateServeceAccount]
     1. IMPORTANT: When you hit save you will see a API Token.. This is the only time you will see it so you need to save it at this point
-    
         ![Alt text][APIToken]
     1. Add the new account to the Role ___Webservice___
 
         ![Alt text][WebServiceRole]
 
-#### Cloud manager
+1. Set Up user for FNMS onPrem:
+    You can use a normal interactive user for the API credentials, but it is recommended to add a special service user for the API connection.
```

Comment:
Add a suggestion to use wstunnel to allow the policy to access the on-premise device.